### PR TITLE
Add inactive focused cell border color

### LINKED ISSUE #114219: Native Notebooks - navigation through notebook subtle
## Steps to reproduce:
1. Open notebook outline
2. Click through the cell headings and see where the selected cell is

When I click through the outline I have a tough time finding where I actually am in the notebook. 
1. If the cell I would like to go to is out of view, when I click it on the outline it doesn't auto-scroll such that the cell is at the top of the notebook canvas. It is somewhere in the middle of the canvas. 
2. When I look for where it is in the canvas, it took me a long time to realize the only indicator was the run button showing up. Even after realizing that was what to look for, it is a bit small and subtle. I was expecting to see the selected state (two blue lines around the cell)

### PR DESCRIPTION
**Pull request by miguelsolorio**
Fixes #114219 and adds an inactive focus border color, `notebook.inactiveFocus`. This is for when your focus is on a cell but your move your focus outside of the editor. Useful when using the new outline view and you click around to navigate. Didn't want to use the `focusBorder` blue color since it would be confusing to have two blue borders. The list widget has a similar color token.

@rebornix @roblourens @jrieken let me know if you have any thoughts on border color choice

https://user-images.githubusercontent.com/35271042/106056203-45af8a80-60a3-11eb-87d0-fc3f30c95c40.mov

### CODE REVIEW

File: src/vs/workbench/contrib/notebook/browser/notebookEditorWidget.ts [hunk 1]
```diff
@@ -2127,6 +2127,12 @@ export const focusedCellBorder = registerColor('notebook.focusedCellBorder', {
 	hc: focusBorder
 }, nls.localize('notebook.focusedCellBorder', "The color of the cell's top and bottom border when the cell is focused."));
 
+export const inactiveFocusedCellBorder = registerColor('notebook.inactiveFocusedCellBorder', {
```
Comment (roblourens) [inline src/vs/workbench/contrib/notebook/browser/notebookEditorWidget.ts:2130]: Just `inactiveFocus` to match other list styles

**Review by roblourens — APPROVED** [review]

**Review by rebornix — APPROVED** [review]

**Comment by roblourens** [comment]
Oh sorry I just meant "focus" instead of "focused". I think it should probably keep "CellBorder", because we could bring back a background or have focus for another notebook element.

**Comment by misolori** [comment]
@roblourens 😆 sorry I misread that, i'll update it in master.

**Comment by misolori** [comment]
@roblourens wait, it seems like we follow the `focused` naming? shouldn't this then be `focused` instead?

image

**Comment by roblourens** [comment]
Oh, if we have already committed to 'focused' in the notebook area then we should keep that. I don't know whether there are already themes depending on these tokens.

**Pull request merged**